formfiller/src/main/java/com/diyasylum/formfiller/application/models/RelativeBuilder.java

```diff
@@ -0,0 +1,20 @@
+package com.diyasylum.formfiller.application.models;
+
+public class RelativeBuilder {
```

Comment:
Did you auto build these somehow?

Reply:
Probably foolishly, I did not. Makes me wish Java had Scala case classes so that lots of this comes for free.

Reply:
Yeah I wouldn't know how. It just looked like maybe you had, and I was curious how you did it.